src/compiler/commandLineParser.ts

```diff
@@ -835,15 +851,17 @@ namespace ts {
             return {
                 options: {},
                 fileNames: [],
-                typingOptions: {},
+                typeAcquisition: {},
                 raw: json,
                 errors: [createCompilerDiagnostic(Diagnostics.Circularity_detected_while_resolving_configuration_Colon_0, [...resolutionStack, resolvedPath].join(" -> "))],
                 wildcardDirectories: {}
             };
         }
 
         let options: CompilerOptions = convertCompilerOptionsFromJsonWorker(json["compilerOptions"], basePath, errors, configFileName);
-        const typingOptions: TypingOptions = convertTypingOptionsFromJsonWorker(json["typingOptions"], basePath, errors, configFileName);
+        // typingOptions has been deprecated. Use typeAcquisition instead.
+        const jsonOptions = json["typeAcquisition"] || json["typingOptions"];

```

Comment:
can you add a comment that we support `typingOptions` only for backward compat purposes and should remove it in a couple of releases